modules/gui/src/com/haulmont/cuba/gui/app/security/role/edit/tabs/ui-permission-tab.xml

```diff
@@ -126,6 +126,11 @@
                 </hbox>
                 <tree id="componentsTree"
                       width="100%">
+                    <actions>
+                        <action id="addComponentIdToComponentTextField"
```

Comment:
1. Too verbose action name. Try something like `pasteComponentId`.
2. A shortcut would be useful
3. An icon would be pretty